index.html

```diff
@@ -21,15 +21,11 @@
         </div>
```

Comment:
- [ ] Please make sure you wrap your hamburger menu in a `<nav>` tag 👍🏽 To make sure your code is semantically correct ✔.

![image](https://user-images.githubusercontent.com/43172164/200025396-2f2084f0-c052-4635-89f5-1bd64fc5f13a.png)